coinflip/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel = "stylesheet" href = "styles.css">
+</head>
+  
+<body>
+    <div class = "container">
+   
+<!-------- Coin div ----------------->
+    <div class = "coin" id = "coin">
+        <div class = "heads">
+    <img src = "images/silverheadscoin.png">
+    </div>
+     <div class = "tails">
+        <img src = "images/goldtailscoin.png">
+    </div>
+   </div>
+   <div class = "stats">
+    <p id = "heads-count"> Heads: 0</p>
+    <p id = "tails-count"> Tails: 0</p>
+</div>
+<div class = "buttons"></div>
+<button id = "flip-button">Flip Coin</button>
+<button id = "reset-button">Reset</button>
+</div>
+</body>
+</html>
+<script src ="script.js"></script>
```

Comment:
This <script> should be in the head tag , if you having trouble with not working, add a defer tag to it . The defer tag will load the load script after the DOM has been parsed. It's essentially the same as what you did by putting the script tag at the end. It should look like `<script src ="script.js" defer></script>`